core/services/gateway/api/message.go

```diff
@@ -126,3 +126,12 @@ func getRawMessageBody(msgBody *MessageBody) [][]byte {
 	copy(alignedReceiver, msgBody.Receiver)
 	return [][]byte{alignedMessageId, alignedMethod, alignedDonId, alignedReceiver, msgBody.Payload}
 }
+
+// SendRequest is used to send a message to the Gateway
+// When the connector receives this request, it selects a receiver to generate a gateway message and signs it
+type SendRequest struct {
```

Comment:
This looks just like a stripped-down MessageBody. Why do we need it? Let's just use MessageBody.

Reply:
sure. original intention was to separate into two immutable objects for clarity:
sendRequest: handlers/capabilities -> connector
message: connector -> gateway

Reply:
I'd rather keep a single object since they are so similar.